source/_integrations/nobo_hub.markdown

```
@@ -50,37 +60,38 @@ This can be utilized the following ways:
+#### No preset "Off"
 
 Nobø heaters do not support preset "Off". This is not a limitation of the integration, but a safety mechanism in the
-Nobø system (perhaps related to frozen pipes due to frost in Nordic regions). 
+Nobø system (perhaps related to frozen pipes due to frost in Nordic regions).
 "Away" temperature is fixed to 7°C and cannot be altered. On/off receivers will be off when the zone is in "Away" status.
 
-To turn heaters completely off, follow these steps (this is a workaround solution): 
+To turn heaters completely off, follow these steps (this is a workaround solution):
+
 1. In the Nobø Energy mobile app, create a week profile.
     - In this profile, set all days to state off. 
```

markdownlint-cli2 (0.17.2)

Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)